src/module/dy.js

```diff
@@ -47,6 +47,15 @@ module.exports = ({ call, getService, logger }) => ({
         throw new Error(`Attributes cannot be undefined: ${undefinedAttrs.join(', ')}`);
       }
     };
+    const validateSortKeyConstraint = (sortKeyConstraint) => {
```

Comment:
This does some validation. Since the object is just spread in `model.entity.query` call I was trying to restrict letting someone pass in a parameter we were not intending like: `parse` or `select`. We could use `Joi-Strict` here instead which would be cleaner.